tests/tracking/fluent/test_fluent.py

```diff
@@ -1498,7 +1498,7 @@ def test_set_experiment_thread_safety(tmp_path):
     origin_create_experiment = MlflowClient.create_experiment
 
     def patched_create_experiment(self, *args, **kwargs):
-        time.sleep(2)
+        time.sleep(0.5)
```

Comment:
why do we need this sleep?

Reply:
With this sleep , we can stably reproduce race conditions which exists in MLflow master. i.e.

the full routine is:
```
1. get experiment by name
2. create experiment if (1) returns None
```
if it is executed in 2 threads or processes in parallel, and we do a sleep before kicking off (2), then the race condition must happen.

if we do not sleep, it probably will be: the first thread finishes (1) and (2), the second thread then start run (1), so it can't stably trigger race condition.

Reply:
Got it. Let's add a comment.